includes/Classifai/Admin/BulkActions.php

```diff
@@ -211,7 +257,11 @@ public function bulk_action_admin_notice() {
 	 * @return array
 	 */
 	public function register_row_action( $actions, $post ) {
-		$post_types = get_supported_post_types();
+		if ( is_a( $this->save_post_handler, '\Classifai\Admin\SavePostHandler' ) ) {
+			$post_types = get_supported_post_types();
+		} elseif ( is_a( $this->embeddings, '\Classifai\Providers\OpenAI\Embeddings' ) ) {
+			$post_types = $this->embeddings->supported_post_types();
+		}
```

Comment:
```suggestion
		$post_types = [];
		if ( is_a( $this->save_post_handler, '\Classifai\Admin\SavePostHandler' ) ) {
			$post_types = array_merge( $post_types, get_supported_post_types() );
		}
		if ( is_a( $this->embeddings, '\Classifai\Providers\OpenAI\Embeddings' ) ) {
			$post_types = array_merge( $post_types, $this->embeddings->supported_post_types() );
		}
```

Currently, if NLU is disabled on the `posts` post type and OpenAPI embeddings is enabled on it then it's not showing classify link in the list table.